src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clone = { ...state };
+  const copyArr = [];
+
+  for (const obj of actions) {
+    if (obj.type === 'addProperties') {
```

Comment:
According to the checklist it's better to use switch statement 🙂

Reply:
agree, let's use switch case here